organization_integration_test.go

```diff
@@ -233,6 +233,28 @@ func TestOrganizationsRead(t *testing.T) {
 			assert.Equal(t, org.DefaultAgentPool.ID, orgAgentTest.DefaultAgentPool.ID)
 		})
 	})
+
+	t.Run("read primary hyok configuration of an organization", func(t *testing.T) {
+		if skipHYOKIntegrationTests {
+			t.Skip()
+		}
+
+		orgName := "" // replace with a valid organization name that has primary hyok configuration
```

Comment:
small nit: I'd lean a little towards making this an [ENV variable](https://github.com/hashicorp/go-tfe/blob/main/docs/TESTS.md#optional-envvars) like some of the other optional tests reference. Just to keep things more consistent.